tests/read_port_mappings_test.py

```diff
@@ -0,0 +1,50 @@
+import os
+import sys
+
+try:
+
+    import sonic_platform_base.sonic_sfp.sfputilhelper
+except Exception as e:
+    print("Failed to load chassis due to {}".format(repr(e)))
+
+
+class TestPortMappingsRead(object):
```

Comment:
For unit tests, we tend to create one file/test class per module we are testing. Maybe we can expand the scope of this file/class to test all the functionality of sfputilhelper. You could rename the `test_port_names()` method to `test_read_port_mappings()`, and we can add more test methods into this file until we have 100% coverage of sfputilhelper.

Reply:
fixed, we can pull in more tests later

Reply:
But can you also rename the file and class `sfputilhelper_test.py` and `TestSfpUtilHelper`, respectively?

Reply:
fixed